app/views/map/index.html.erb

```diff
@@ -1 +1,243 @@
+
+		// // grouping buildings by campus
+		// var buildings_campus_I = L.layerGroup([main_building, lecture_hall, a_building, b_building, c_building, j_house]);
+		// var buildings_campus_II = L.layerGroup([e_house, f_house, villa]);
+		// var buildings_campus_III = L.layerGroup([g_house]);
+		
+		// // setting markers for interesting eat-in places
+		// var ulf = L.marker([52.39333033686638, 13.131310644592734],{opacity: 0.0}).bindTooltip('Ulf\'s coffee', {permanent: true, className: "marker_label", direction: "center"}),
+		// 	mensa_griebnitzsee = L.marker([52.393092597902964, 13.129241382236938],{opacity: 0.0}).bindTooltip('Mensa <br> Griebnitzsee', {permanent: true, className: "marker_label", direction: "center"}),
+		// 	mensa_movie = L.marker([52.38983920408487, 13.116905316298654],{opacity: 0.0}).bindTooltip('Movie Mensa', {permanent: true, className: "marker_label", direction: "center"});
+
+		// // grouping eat-in places
+		// var eating = L.layerGroup([ulf, mensa_griebnitzsee, mensa_movie]);
+
+		// // adding everything to an overlay
+		// var overlay_poi = {
+    	// 	"Campus I": buildings_campus_I,
+		// 	"Campus II": buildings_campus_II,
+		// 	"Campus III": buildings_campus_III,
+		// 	"Eating": eating
+		// };
+
+		// labels = L.layerGroup([buildings_campus_I, buildings_campus_II, buildings_campus_III, eating]);
+		// labels.eachLayer(function (layer) {
+		// 	if (!mymap.hasLayer(layer)){
+		// 		mymap.addLayer(layer);
+				
+		// 	}
+		// });
+
+
+		// // add labels only when zoomed in
+		// mymap.on('zoomend', function() {
+		// var zoomlevel = mymap.getZoom();
+		// 	if (zoomlevel > 16 && zoomlevel < 20){
+		// 		// these is the range where we want to show the text
+		// 		labels.eachLayer(function (layer) {
+		// 			if (mymap.hasLayer(layer)){
+		// 				console.log("layer already added");
+		// 			} else {
+		// 				mymap.addLayer(layer);
+		// 			}
+		// 		})
+		// 	}
+		// 	else {
+		// 		labels.eachLayer(function (layer) {
+		// 			if (mymap.hasLayer(layer)){
+		// 				mymap.removeLayer(layer);
+		// 			} else {
+		// 				console.log("layer already added");
+		// 			}
+		// 		})
+		// 	}
+		// //console.log("Current Zoom Level =" + zoomlevel)
+		// });
+
+
+
+
+		// // control font sizes
+		// mymap.on('zoomend', function () {
+		// 	var zoomLevel = mymap.getZoom();
+		// 	var tooltip = document.querySelector(".leaflet-tooltip");
+
+		// 	console.log(zoomLevel);
+		// 	switch (zoomLevel) {
+		// 		case 17:
+		// 			tooltip.style.fontSize = "3pt";
+		// 			break;
+		// 		case 18:
+		// 			tooltip.style.fontSize = "50pt";
+		// 			break;
+		// 		case 19:
+		// 			tooltip.style.fontSize = "18px";
+		// 			break;
+		// 		case 20:
+		// 			tooltip.style.fontSize = "20px";
+		// 			break;
+		// 		case 21:
+		// 			tooltip.style.fontSize = "20px";
+		// 			break;
+		// 		case 22:
+		// 			tooltip.style.fontSize = "22px";
+		// 			break;
+		// 		default:
+		// 			tooltip.style.fontSize = "14px";
+		// 			break;
+		// 	}
+		// });
```

Comment:
I'd either un-comment this block, or remove it. We should not have huge chunks of commented out code. If we keep these lines, we should refactor so that not all map-related stuff sits in this .html.erb. We will need to do that anyways, eventually

Reply:
done by merging latest commits from `map3`